examples/common/config.yaml

```diff
@@ -3,38 +3,54 @@
 # https://www.envoyproxy.io/docs/envoy/latest/configuration/configuration
 static_resources:
   listeners:
-    - address:
-        socket_address: {address: 0.0.0.0, port_value: 9001, protocol: TCP}
-      filter_chains:
-        - filters:
-            - name: envoy.http_connection_manager
-              config:
-                stat_prefix: client
-                route_config:
-                  virtual_hosts:
-                    - name: all
-                      domains: ["*"]
-                      routes:
-                        - match: {prefix: "/"}
-                          route: {cluster: hello_world_api}
-                http_filters:
-                  - name: envoy.router
+  - address:
+      socket_address:
+        protocol: TCP
+        address: 0.0.0.0
+        port_value: 9001
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
+          stat_prefix: ingress_http
```

Comment:
nit: change this to back to "client" imo ("ingress_http" sounds a lot like a proxy, which technically this still is, but is increasingly misleading).

Reply:
Good call, fixed